mod/carrierwave/lib/carrier_wave/file_card_uploader.rb

```diff
@@ -232,16 +236,16 @@ def storage
   end
 
   class SanitizedFile
-    def content_type
-      # the original content_type method doesn't seem to be very reliable
-      # It uses mime_magic_content_type  - which returns invalid/invalid for css files
-      # that start with a comment - as the second option.  (we switch the order and
-      # use it as the third option)
-      @content_type ||=
-        existing_content_type ||
-        mini_mime_content_type ||
-        mime_magic_content_type
-    end
+    # def content_type
+    #   # the original content_type method doesn't seem to be very reliable
+    #   # It uses mime_magic_content_type  - which returns invalid/invalid for css files
+    #   # that start with a comment - as the second option.  (we switch the order and
+    #   # use it as the third option)
+    #   @content_type ||=
+    #     # existing_content_type ||
+    #     mini_mime_content_type ||
+    #     mime_magic_content_type
+    # end
```

Comment:
This method completely broke, as none of the methods it called exist any more. But I'm not very confident that the correct content types are still found.

Reply:
mini_mine was replaced with marcel https://github.com/carrierwaveuploader/carrierwave/pull/2552

Reply:
This is the new approach: https://github.com/carrierwaveuploader/carrierwave/blob/d27c9871c83613717bd69d31da7d23450bf266c9/lib/carrierwave/sanitized_file.rb#L241
Maybe it works now and we don't have to override it anymore.